getting_started/scaling-your-cluster.adoc

```diff
@@ -0,0 +1,10 @@
+[id="scaling-your-cluster"]
+= Scaling your cluster
+include::modules/common-attributes.adoc[]
+:context: access
+
+toc::[]
+
+include::modules/recommended-node-host-practices.adoc[leveloffset=+1]
```

Comment:
We had talked about just stating what the result of how this is configured for OSD since the customer cannot configure this themselves. I checked and OSD clusters do not have any customization on these values.

Given that, I think we can get rid of it altogether. I dont know what the default OCP limits are for this setting, but since the OCM docs don't state it and it is probably much higher than any OSD customer will do, the risk is low.

Reply:
Agreed - removed node host practices.